bedrock/firefox/views.py

```diff
@@ -636,12 +636,15 @@ def download_thanks(request):
     locale = l10n_utils.get_locale(request)
     variant = request.GET.get('v', None)
     show_newsletter = locale in ['en-US', 'en-GB', 'en-CA', 'en-ZA', 'es-ES', 'es-AR', 'es-CL', 'es-MX', 'pt-BR', 'fr', 'ru', 'id', 'de', 'pl']
-    pre_download_locales = ['en-US', 'en-GB', 'en-CA', 'en-ZA', 'de', 'fr']
+    pre_download_locales = ['fr']
 
     # ensure variant matches pre-defined value
-    if variant not in ['a']:  # place expected ?v= values in this list
+    if variant not in ['a', 'b', 'c', 'd']:  # place expected ?v= values in this list
```

Comment:
Where is the `d` coming from?

Reply:
Leftover from when there we're 4 variants (one was scrapped), will update!